### ISSUE TITLE
Add support for custom regex for remote patterns

### ISSUE THREAD

**Issue by trevorhauter** [trevorhauter/gitportal.nvim#69]
Here's an example for a similar plugin:  https://github.com/folke/snacks.nvim/blob/main/docs/gitbrowse.md?

This was suggested here: https://github.com/trevorhauter/gitportal.nvim/issues/64#issuecomment-2531555292

This will be a huge help for people with special use cases, or to allow them to use remotes that aren't yet supported!

**Comment by KamilKleina** [comment]
Any chance you know if option to use the plugin with Forgejo, Onedev etc are coming soon?

**Comment by trevorhauter** [comment]
@KamilKleina Yup! I'm resuming work on the plugin this month, I took a couple week hiatus from programming over the end of the year to recharge. Which host would you prefer to be available first?

**Comment by KamilKleina** [comment]
If it was up to me, forgejo is something that I use in a lot of projects that specially open source so I guess if it was up to me I would start there,

Awesome to hear btw!

**Comment by trevorhauter** [comment]
This project is moving over to codeberg. I'm going to close this issue here. If this still seems useful to you, please [open an issue](https://codeberg.org/trevorhauter/gitportal.nvim/issues) over there!
